uk_bin_collection/uk_bin_collection/councils/CastlepointDistrictCouncil.py

```
@@ -53,68 +55,57 @@ def parse_data(self, page: str, **kwargs) -> dict:
+                data["bins"].append(dict_data)
+            for day in normal_days:
+                dict_data = {
+                    "type": "Normal collection",
+                    "collectionDate": datetime(
+                        year=int(year), month=int(month), day=int(day)
+                    ).strftime(date_format),
+                }
+                data["bins"].append(dict_data)
+
+        data["bins"].sort(
+            key=lambda x: datetime.strptime(x.get("collectionDate"), "%d/%m/%Y")
```

Ruff (0.14.10)

`datetime` may be undefined, or defined from star imports

(F405)